databases/week2/more.queries.sql

```diff
@@ -0,0 +1,33 @@
+USE nnn;
+
```

Comment:
The queries look good, but i think you need to use different tables and data for this part, according to the instructions.